turingschool-examples/fitlit-starter-kit
# Feat/second sleep chart

#### What's this PR do?
- This will add a second chart that displays the sleep quality over one week for a user.
#### Where should the reviewer start?
- Be sure to run `npm install chart.js` if you haven't already
- Effected files are Chart.js, scripts.js, and index.html
#### How should this be manually tested?
- Run `npm start` and navigate to http://localhost:8080/ in your browser.  You should see two charts displayed
#### Any background context you want to provide?
n/a
#### What are the relevant tickets
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
